types/enginko/EGK-LW20W00/distance.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://akenza.io/enginko/EGK-LW20W00/distance.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Distance",
```

Comment:
fill_level might be more accurate as the topic name, what do you think?

Reply:
Filllevel is only accurate if its configured, so i think distance would be a save bet.

Reply:
Distance is only one of three right? so it might not be the best name